### PR TITLE
REF: upgrade bech32, bip39 and crypto-js

### PR DESCRIPTION
**Pull request by limpbrains**
I've looks through BW dependencies and tried to bump ones related to crypto or bitcoinjs.

* assert - dropped old IE support
* bech32 - new way to import `bech32.decode` function
* bip39 - dropped `mnemonicToSeedHex`, renamed `mnemonicToSeed` to `mnemonicToSeedSync`
* crypto-js - migrated from `Math.random()` to crypto dependency. Since we are not using it to generate keys it is safe to upgrade

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5km2r3q3cfkn1du6456r36xmjr?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/limpbrains-upgrade-bip39-1620556616.apk)

File: package.json [hunk 1]
```diff
@@ -82,29 +82,29 @@
...
-    "bip39": "2.6.0",
+    "bip39": "3.0.4",
     "bitcoinjs-lib": "5.2.0",
     "bitcoinjs-message": "2.2.0",
     "bolt11": "1.3.1",
     "buffer": "6.0.3",
     "buffer-reverse": "1.0.1",
     "coinselect": "3.1.12",
-    "crypto-js": "3.1.9-1",
+    "crypto-js": "4.0.0",
```
Comment (Overtorment) [inline package.json:107]: last time i tried to bump `crypto-js` my tests for aes encrypt/decrypt started failing

Comment (Overtorment) [reply package.json:107]: weird i dont see unit tests failing now...

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8v8w020agv6vd8fy4ckd5r5h48?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/limpbrains-upgrade-bip39-1620571236.apk)

File: tests/unit/encryption.test.js [hunk 2]
```diff
@@ -34,4 +34,12 @@ describe('unit - encryption', function () {
     );
     assert.ok(!decrypted);
   });
+
+  it('can decrypt cipher created by CryptoJS@3.1.9-1', () => {
```
Comment (Overtorment) [inline tests/unit/encryption.test.js:38]: noice

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by limpbrains** [comment]
@GladosBlueWallet bad bot!

**Comment by Overtorment** [comment]
okay, build worked fine on my ios device via testflight.
dont know what else we can test, all seems good

**Pull request merged**